deposit/src/main/java/edu/unc/lib/deposit/work/DepositSupervisor.java

```diff
@@ -599,12 +598,12 @@ private Job getNextJob(Job job, String depositUUID, Map<String, String> status,
         }
 
         // Perform vocabulary enforcement for package types that retain the original metadata
-        if ((packagingType.equals(PackagingType.METS_DSPACE_SIP_1.getUri())
-                || packagingType.equals(PackagingType.METS_DSPACE_SIP_2.getUri())
-                || packagingType.equals(PackagingType.PROQUEST_ETD.getUri()))
-                && !successfulJobs.contains(VocabularyEnforcementJob.class.getName())) {
-            return makeJob(VocabularyEnforcementJob.class, depositUUID);
-        }
+//        if ((packagingType.equals(PackagingType.METS_DSPACE_SIP_1.getUri())
```

Comment:
Yeah, once we have a vocabulary manager implementation we should be able to turn it back on